andela/mrm-mobile
# #160140608 Fix Schedule Timeline Strip Issue

**What does this PR do?**
Fix schedule timeline strip issue

**Description of Task to be completed?**
The schedule timeline strip should show only from 8 am to 5 pm

**How should this be manually tested?**
- Run the application.
- Select a meeting room from a list of rooms.
- The timeline strip should display correctly from 8 am to 5 pm.

**Any background context you want to provide?**
Previously, The schedule timeline strip showed from 8 am to 9 pm and a user had to scroll sideways to see the end of the strip.

**What are the relevant pivotal tracker stories?**
#160140608

**Screenshots:**
_7" tablet_
<img width="849" alt="screen shot 2018-09-05 at 11 50 21" src="https://user-images.githubusercontent.com/33781312/45082405-eaf7eb80-b101-11e8-92aa-75d65cfe1dff.png">

_10" tablet_
<img width="755" alt="screen shot 2018-09-07 at 09 47 33" src="https://user-images.githubusercontent.com/33781312/45202917-2110a900-b283-11e8-87bf-8e0cccf49353.png">
